components/developer/ccache/files/ccache-update-symlinks.sh

```diff
@@ -109,16 +109,28 @@ if [ "$NAMES" = "$LINKS" ]; then
 fi
 
 # Link missing compilers to get wrapped by ccache for its consumers
+# Note: originally used `fgrep` but it misfired for substring matches
+# finding e.g. "gcc" or "gcc-4" in "gcc-4.9"; not a full-string match
 for N in $NAMES ; do
-    echo "$LINKS" | fgrep "$N" >/dev/null || \
```

Comment:
will `grep -w` work?

Reply:
Not quite, `gcc` will be found in `gcc-10` for example